docs/reference/primitives/app/timer.md

```diff
@@ -28,6 +28,7 @@ A Timer is defined in [`convox.yml`](/configuration/convox-yml).
 | **command**  | **yes**  | The command to execute once the [Process](/reference/primitives/app/process) starts                               |
 | **schedule** | **yes**  | A cron formatted schedule for spawning the [Process](/reference/primitives/app/process). All times are UTC        |
 | **service**  | **yes**  | The name of the [Service](/reference/primitives/app/service) that will be used to spawn the [Process](/reference/primitives/app/process) |
+| **concurrencyPolicy**  | **no**  | It specifies how to treat concurrent executions of a job that is created by this cron job. Check this [doc](https://kubernetes.io/docs/tasks/job/automated-tasks-with-cron-jobs/#concurrency-policy) for more info. |
```

Comment:
this is a bit of a nit but could we change this param to `concurrency` only? just to keep it short just like the other parameters? Also, I think the default should be `forbid` instead of `no` (IIRC `forbid` it what we use by default in the Backend). We should probably also allow lowercase options (e.g `forbid`) as I don't think we enforce Capitalize parameters anywhere in `convox.yml`

#nit2 the MD table formatting looks a bit off

Reply:
hey, @heronrs, in this case, `no` is not the default value but if this field is required.

Reply:
oh lol ok thanks, my bad